exercises/yacht/stack.yaml

```
@@ -0,0 +1 @@
+resolver: lts-14.0
```

We may want to update the resolver across all exercises, but I think there is some advantage to having the same resolver for every exercise; in particular CI builds trigger the same compiler and library downloads.
```suggestion
resolver: lts-12.4
```